.github/actions/rerun-job-action/action.yml

```diff
@@ -0,0 +1,118 @@
+name: "Rerun Job Action"
```

Comment:
I think this will need the Apache header (can copy/paste from any workflow).

Also, could you add a brief comment description of the purpose of this composite action, including how to use it (ideally with a code snippet) and required permissions (again ideally with a code snippet). I think the permissions should be `repo: read` (even if we don't need this, virtually every job will), `pull-requests: write`, and `checks: read` (might actually need to be `statuses` instead of `checks`?)

Reply:
Forgot the header, Will add it. 

Regarding permissions i was thinking to add that and explain in some doc showing implementation of the action. Or i can do it here. Put README.MD here?

Reply:
I'm fine leaving it for the doc. When we add that, we should add a link from here. I'd still like to have a basic 1 sentence summary of this action in the file though.